### PR TITLE
Fixes #265

### LINKED ISSUE #265: Undefined index notices thrown by facebook-commerce-pixel-event.php
The changes made to support reverse proxy, are throwing the following notices
```
PHP Notice:  Undefined index: pixel_render in /home/XXXXXX/public_html/wp-content/plugins/facebook-for-woocommerce/facebook-commerce-pixel-event.php on line 85
PHP Notice:  Undefined index: proxy_basecode in /home/XXXXXX/public_html/wp-content/plugins/facebook-for-woocommerce/facebook-commerce-pixel-event.php on line 225
PHP Notice:  Undefined index: no_script_render in /home/XXXXXX/public_html/wp-content/plugins/facebook-for-woocommerce/facebook-commerce-pixel-event.php on line 158
```

### PR DESCRIPTION
**Pull request by glagonikas**
Fixes #265

### CODE REVIEW

**Comment by glagonikas** [comment]
Hey @jlkalberer considering this PR overwrites part of your previous commit, it's probably worth reviewing it.

File: facebook-commerce-pixel-event.php [hunk 1]
```diff
@@ -222,12 +228,11 @@ public static function set_basecode($new_basecode) {
 
   public static function get_basecode() {
     $fb_options = self::get_options();
-    $basecode = $fb_options[self::BASECODE_KEY];
-    if (!isset($basecode)) {
+    if (!isset($fb_options[self::BASECODE_KEY])) {
```
Comment (dmitridr) [inline facebook-commerce-pixel-event.php]: We rewrote this function entirely and change it to just return the basecode. So you'll want to undo the changes here.
